src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,141 @@
+                    }
+                  """
+             Then it should respond with 201
+              And the record can be retrieved with a GET request
+                            # TODO: when writing the step definition,
+                            # be sure to validate that the response matches the original payload,
+                            # except for the addition of the 'id' field
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "",
+                        "description": "Wrong Value",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Wrong Value"
+                    }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                        "$.codeValue": [
+                            "CodeValue is required and should not be left empty."
+                        ]
+                  }
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data Forbidden
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "",
+                        "description": "Wrong Value",
+                        "namespace": "uri://.org/wrong",
+                        "shortDescription": "Wrong Value"
+                    }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                        "$.codeValue": [
+                                "CodeValue is required and should not be left empty."
+                            ]
+                        }
+                    }
```

Comment:
Accidentally copied the message from above instead of the invalid namespace message:

```json
{
  "detail": "Access to the resource could not be authorized. The 'Namespace' value of the resource does not start with any of the caller's associated namespace prefixes ('uri://ed-fi.org').",
  "type": "urn:ed-fi:api:security:authorization:namespace:access-denied:namespace-mismatch",
  "title": "Authorization Denied",
  "status": 403,
  "correlationId": "de6f3b55-4a0b-4fc2-9150-aef03fc89b18"
}
```

Reply:
This response still needs to be updated. And the code on line 76 should be 403.

Reply:
I misunderstood, and I was getting the 400 because the codeValue was empty. I added the right message

```
             Then it should respond with 403
              And the response body is
                  """
                    {
                        "detail": "Access to the resource could not be authorized. The 'Namespace' value of the resource does not start with any of the caller's associated namespace prefixes ('uri://ed-fi.org', 'uri://gbisd.org', 'uri://tpdm.ed-fi.org').",
                        "type": "urn:ed-fi:api:security:authorization:namespace:access-denied:namespace-mismatch",
                        "title": "Authorization Denied",
                        "status": 403,
                        "correlationId": null
                    }
```